### DISCUSSION TITLE
Architecture Specific Cask

### DISCUSSION THREAD

**Discussion by mys721tx** [Homebrew/discussions#3096] [Tap maintenance and Homebrew development]
Hello all,

Some recent versions of [VMware Fusion technical preview](https://customerconnect.vmware.com/downloads/get-download?downloadGroup=FUS-PUBTP-2021H1) are specific to ARM64 only. Since there is no equivalent version of TP 19431034 for Intel, should a new ARM64 package be added? Should we add some mechanism in the future to provide architecture specific casks?

**Comment by SMillerDev** [comment]
You can declare it depends on `arch: :arm` or you can add an if statement to the current version that does version a if x86 and b otherwise.
